sympy/utilities/lambdify.py

```diff
@@ -151,10 +149,20 @@ def _import(module, reload="False"):
     for sympyname, translation in translations.items():
         namespace[sympyname] = namespace[translation]
 
+    # For computing the modulus of a sympy expression we use the builtin abs
+    # function, instead of the previously used fabs function for all
+    # translation modules. This is because the fabs function in the math
+    # module does not accept complex valued arguments. (see issue 9474). The
+    # only exception, where we don't use the builtin abs function is the
+    # mpmath translation module, because mpmath.fabs returns mpf objects in
+    # contrast to abs().
```

Comment:
Maybe you should open an mpmath issue to change this. `abs` can be overridden with `__abs__` (which is how `abs(x)` -> `Abs(x)` works for SymPy objects).